probe/src/probemon.py

```diff
@@ -44,6 +48,8 @@ def __init__(self, size):
         self.vendor = LRU(size)
 
 def print_fields(fields):
+    if fields[1] not in arr:
+        arr.append(fields[1])
```

Comment:
Thinking about it now, you could instead use the [set](https://docs.python.org/3.8/library/stdtypes.html#set-types-set-frozenset) data structure which contains only unique items, a la `arr = set()`, then

```suggestion
    arr.add(fields[1])
```

So if the MAC address is already known, it won't be added twice. But really just a nitpick.